src/shared/tar-util.c

```diff
@@ -124,6 +125,9 @@ static int open_inode_apply_acl(OpenInode *of) {
         }
```

Comment:
If ACLs are supported, we return an error if they could not be applied succesfully. If a build is made without acl support, I think we should at least warn if any ACLs would have been applied.